src/Modal/index.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './styles.css';
+import Button from '../Button';
+
+function Modal({ isOpen, onCancel, onConfirm }) {
+  return (
+    <div className={`modal ${isOpen ? 'open' : ''}`}>
+      <h2>Confirm Reset</h2>
+      <Button onClick={onConfirm} label="CONFIRM" />
```

Comment:
```suggestion
      <h2>Clear Deck?</h2>
      <Button onClick={onConfirm} label="Yes" />
```

A little bit of clarification on what the action is doing.
Pose it as a question.